text/0000-advance-security-disclosure.md

```diff
@@ -0,0 +1,305 @@
+information about those vulnerabilities confidential before public disclosure.
+
+## Security releases pre-announcements
+
+To ensure all Rust users are aware of upcoming security releases, before public
+disclosure the Security Response WG will publish a pre-announcement of the
+upcoming release on the [public security announces mailing list][announce-ml]
+and the [Rust blog][blog], mentioning the planned date and time of the release,
+the severity of the vulnerability and the affected part of the project.
+
+This will allow all Rust users to schedule the time to update their toolchain.
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+## Advance disclosures
+
+### List of notified organizations
+
+The Rust Security Response WG will maintain a list of organizations who are
+eligible to receive advance disclosures. Organizations (either companies or
+established open source projects) matching any of these criteria can apply:
+
+* Organizations distributing the Rust toolchain (as provided by the Rust
+  project) or a fork of the Rust toolchain explicitly meant to be used by their
+  external customers or users. Organizations shipping the toolchain just to
+  internal customers/users are not eligible, nor are organizations publicly
+  shipping a toolchain primarily meant for internal use.
+
+* Organizations developing an independent implementation of the Rust language,
+  not originating from a fork of the toolchain provided by the Rust project.
+  The implementation has to be mature, widely used and must follow all policies
+  on alternate implementation set by the Rust project (if any are set).
+
+Note that when evaluating the eligibility criteria, the spirit of the rules
+will be considered: if the WG thinks an organization that would not be eligible
+otherwise is trying to find a loophole to be able to apply, the WG will have
+the authority to reject the application.
+
+Organizations will be able to apply by opening an issue in the
+[rust-lang/wg-security-response] repository, detailing:
+
+* Which of their products or projects would make them eligible to be notified.
+
+* One or more email addresses that should receive the advance disclosure.
+
+* Two or more people that will serve as the organizational contact: those
+  people will be contacted by the WG whenever the WG needs to discuss something
+  with the organization.
+
+* A pledge to maintain confidentiality, as detailed in the "Confidentiality"
+  section of this RFC. The person submitting the application must be authorized
+  to enter that pledge on behalf of their organization.
+
+The application process will be public, and the decision will be made by the
+WG: a majority of the WG members needs to be in favor of the organization
+joining, and none of the members needs to be against it. The resulting list
+of notifications receiving advance disclosure will also be public.
```

Comment:
I think this should be "organizations".

```suggestion
joining, and none of the members needs to be against it. The resulting list
of organizations receiving advance disclosure will also be public.
```

cc @xFrednet if that's the case, this is the answer to your first question :)

Reply:
Yep, that's a typo on my end, it should be "organizations" here. I'll fix it later.